packages/context/src/binding.ts

```diff
@@ -72,6 +94,10 @@ export class Binding {
     return this._tags;
   }
 
+  get scope() {
```

Comment:
What is the benefit of having a private property `_scope` and then a public getter + setter (called `inScope`), instead of providing a public property `scope` accompanied by a syntactic sugar method `inScope()`?